### DISCUSSION TITLE
Auslegeordnung zur E-ID auf dem Winterkongress 2022 der Digitalen Gesellschaft

### DISCUSSION THREAD

**Discussion by christianheimann** [e-id-admin/governance-sounding-board#26] [Show and tell]
@fforootd schafft es, in diesem Video-Beitrag die Komplexität und Vielseitigkeit der Problematik zur E-ID weiter auszulegen:
https://www.youtube.com/watch?v=eVAA0HJ-rOI  
Danke für diesen Beitrag zur E-ID Diskussion! 

Als Antwort auf eine wichtige Aussage ist sicher die [Auswertung der im Herbst durchgeführten informellen Konsultation](https://www.bj.admin.ch/dam/bj/de/data/staat/gesetzgebung/staatliche-e-id/ber-konsultation.pdf.download.pdf/ber-konsultation-d.pdf) dienlich, aber nicht ganz so deutlich:

> "Die E-ID sollte primär den Zweck eines elektronischen Ausweises verfolgen."

Das war auch die Meinung der grossen Mehrheit in der Konsultation. Und dieser Meinung ist das Projektteam E-ID bestrebt, gerecht zu werden. Die im Zielbild definierte Vision **"Eine E-ID ist ein vom Staat ausgestellter digitaler Ausweis, um die eigene Identität nachweisen zu können"** wird verfolgt. Ich kann aber nachvollziehen, dass hierzu Unklarheiten auftauchen können, da in der Kommunikation dieser simple aber wichtige Fakt bisher nicht oft in den Vordergrund getreten ist.

**Comment by fforootd** [comment]
@christianheimann danke für die Erwähnung hier 😁 


> "Die E-ID sollte primär den Zweck eines elektronischen Ausweises verfolgen."

Ich denke genau das sollte klar kommuniziert werden damit Missverständnisse verhindert werden.
